modules/managing-namespace-auto-pruning-policies.adoc

```diff
@@ -488,10 +414,11 @@ $ curl -X POST \
     "method": "<creation_date>",
     "value": "<7d>",
     "tagPattern": "<^test.>*",
-    "tagPatternMatches": <true>
+    "tagPatternMatches": <false> <1>
   }' \
   "https://<quay-server.example.com>/api/v1/repository/<organization_name>/<repository_name>/<tag_name>/autoprunepolicy/"
```

Comment:
Should not have `<tag_name>` here. 
Should be 
```
"https://<quay-server.example.com>/api/v1/repository/<organization_name>/<repository_name>/autoprunepolicy/"
```

Reply:
Fixed.